### ISSUE TITLE
Common default cookie manager

### ISSUE THREAD

**Issue by AntonioSun** [smicyk/groovy-jmeter#70]
How to set a default cookie manager to all my requests?

**Comment by smicyk** [comment]
Yes, I believe so. In this, it behaves the same as in JMeter GUI, If you put it under the group each thread from this group would have its own cookie storage area and each request should use it.

**Comment by AntonioSun** [comment]
Thanks while I have you, if I put https://github.com/smicyk/groovy-jmeter/wiki/reference#headers under the group, just like the `cookies()` above, with some default values, like Host, Origin, Referer, Connection, User-Agent etc, 

Would each thread from this group and each request use such default values, together with the headers specified for themself?

**Comment by AntonioSun** [comment]
Hi @smicyk,

How about 

          check_response {
            status() eq 200
          }

Does it has to go with each request, or I can place one at group level just like the `cookies()` above?

What if I want to check **_each_** response for keyword "`Wrong`" and mark the request fail if found. How can I do that?

**Comment by smicyk** [comment]
If you put check_response on group level it will be added to all request (the assertion will check each request during execution). Please note, that you can't override assertion if you would like to have some different check on one particular request e.g.

```groovy
plan {
   group {
       http 'GET /books'
       http 'GET /authors', {
           check_response {
               status() eq 404
           }
       }
       check_response {
           status() eq 200
       }
}
```
Then the /books will be asserted only with __status eq 200__  and /authors will be asserted with __status eq 200__ and __status eq 404__. So one of the will fail.
